### ISSUE TITLE
Link coloring for accessibility

### ISSUE THREAD

**Issue by MrPackman4019** [BanklessDAO/bankless-dao-ghost-website#73]
Need to review design and decide how links should be colored differently than normal text for accessibility.

Need to add screenshot of where this issue occurs on the site currently,
